src/sprite/mod.rs

```diff
@@ -352,6 +352,10 @@ impl SpritesheetBuilder {
         self
     }
 
+    /// Adds the metadata that all images are sdf sprites.
+    /// 
+    /// You have to ensure that the Sprites are created as a sdf file beforehand.
+    /// See [`Sprite::new_sdf`] for further context.
```

Comment:
```suggestion
    /// Add metadata to indicate that all images are SDF sprites.
    ///
    /// You have to ensure that the sprites are created as an SDF file beforehand. See
    /// [`Sprite::new_sdf`] for further context.
```

Reply:
> Do you mind if I ask for a few tweaks

This is your repository xD

Sorry about the line lenghts, I just did it quick-and-dirty in githubs editor